cereal/log.capnp

```diff
@@ -2479,6 +2479,13 @@ struct Microphone {
   filteredSoundPressureWeightedDb @2 :Float32;
 }
 
+struct AudioData {
+  data @0 :Data;
+  sampleRate @1 :UInt32;
+  startFrameIdx @2 :UInt64;
+  sequenceNum @3 :UInt64;
```

Comment:
changed my mind on this, let's just delete both since we're also going to put it in the qcam. we can add later if we need it